x-pack/metricbeat/module/aws/_meta/config.yml

```diff
@@ -30,11 +30,16 @@
     - sns
     - sqs
 - module: aws
-  period: 12h
+  period: 24h
   metricsets:
     - billing
-  regions:
-    - us-east-1
+  cost_explorer_config:
+    group_by_dimension_keys:
```

Comment:
I don't know if this kind of settings are documented somewhere but I miss a comment giving some info about them. wdyt?

Reply:
Just added 😄 Thanks for catching it!!